packages/explorable-explanations/src/protectedAudience/modules/auctions/index.js

```diff
@@ -104,6 +104,19 @@ auction.draw = (index) => {
     app.promiseQueue.push(async (cb) => {
       const { component, props, callBack, delay = 0 } = step;
 
+      if (props?.title) {
+        const deciderObject = {
+          title: props.title,
+          description: props.description,
+          ssp: props?.ssp
+            ? props.ssp
+            : app.isMultiSeller && props.title === 'scoreAd()'
+            ? 'https://www.' + config.timeline.circles[index].website
+            : '',
```

Comment:
Please avoid complex nested ternary operator 🙏

Reply:
Resolved in latest commits.